src/bezpath.rs

```diff
@@ -96,8 +111,28 @@ impl VelloVector for VelloBezPath {
 
         path
     }
+
+    fn border_translation(&self, time: f64) -> DVec2 {
+        let (path, t) = self.inbetween(time);
+
+        let current = path[0].end_point().unwrap_or_default();
+        let point = interp_pathel(current, path[path.len() - 1], t as f32).end_point().unwrap().to_vec2();
+
+        DVec2::new(point.x, point.y)
+    }
+
+    fn border_rotation(&self, time: f64) -> f64 {
+        let (path, t) = self.inbetween(time);
+
+        let current = path[0].end_point().unwrap_or_default();
+        let point = interp_pathel(current, path[path.len() - 1], t as f32).end_point().unwrap().to_vec2();
+        let current = current.to_vec2();
+
+        DVec2::normalize_or_zero(DVec2::new(point.x, point.y) - DVec2::new(current.x, current.y)).to_angle()
+    }
 }
 
+/// Interpolate [`kurbo::PathEl`].
 fn interp_pathel(p0: kurbo::Point, pathel: kurbo::PathEl, t: f32) -> kurbo::PathEl {
```

Comment:
could dramatically [simplify this](https://acegikmo.medium.com/the-ever-so-lovely-b%C3%A9zier-curve-eb27514da3bf) but im not sure if we should sacrifice the readability. also any particular reason why the time here is `f32` instead of the norm `f64`?

also i still think this should be in upstream kurbo but its whatever